resources-domain/lambdas/import-producer/src/khpMappings.ts

```diff
@@ -250,6 +249,8 @@ const KHP_MAPPING_NODE_TAXONOMIES: { children: MappingNode } = {
 };
 
 export const KHP_MAPPING_NODE: MappingNode = {
+  _id: resourceFieldMapping('id'),
+  // TODO: Remove this and all other 'objectId' mappings once the updated Arctic API is deployed to production
   objectId: resourceFieldMapping('id'),
```

Comment:
This just adds two properties with the same value? Asking to know if there's any risk of "overriding actual id with null/undefined".

Reply:
Yes, exactly, it just duplicates the value to another output property